gradle/libs.gradle

```diff
@@ -1,4 +1,5 @@
 project.ext.libs = [
+    'bcpkix': 'org.bouncycastle:bcpkix-jdk15on:1.54',
```

Comment:
I'm a bit concerend about taking a bouncycastle dependency, can we assume this is provided if you want to use PCKS#1 certs and just have good errors if it's not available?